ReleaseTooling/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -666,78 +715,43 @@ struct ZipBuilder {
-          try ResourcesManager.packageAllResources(containedIn: podInfo.installedLocation)
-        } catch {
-          fatalError("Tried to package resources for \(podName) but it failed: \(error)")
-        }
+    // Copy each of the frameworks to a known temporary directory and store the location.
+    for framework in podInfo.binaryFrameworks {
+      // Copy it to the temporary directory and save it to our list of frameworks.
+      let zipLocation = binaryZipDir.appendingPathComponent(framework.lastPathComponent)
+      let carthageLocation =
+        binaryCarthageDir.appendingPathComponent(framework.lastPathComponent)
 
-        // Copy each of the frameworks to a known temporary directory and store the location.
-        for framework in podInfo.binaryFrameworks {
-          // Copy it to the temporary directory and save it to our list of frameworks.
-          let zipLocation = binaryZipDir.appendingPathComponent(framework.lastPathComponent)
-          let carthageLocation =
-            binaryCarthageDir.appendingPathComponent(framework.lastPathComponent)
-
-          // Remove the framework if it exists since it could be out of date.
-          fileManager.removeIfExists(at: zipLocation)
-          fileManager.removeIfExists(at: carthageLocation)
-          do {
-            try fileManager.copyItem(at: framework, to: zipLocation)
-            try fileManager.copyItem(at: framework, to: carthageLocation)
-          } catch {
-            fatalError("Cannot copy framework at \(framework) while " +
-              "attempting to generate frameworks. \(error)")
-          }
-          frameworks.append(zipLocation)
-
-          CarthageUtils.generatePlistContents(
-            forName: framework.lastPathComponent.components(separatedBy: ".").first!,
-            withVersion: podInfo.version,
-            to: carthageLocation
-          )
-          carthageFrameworks.append(carthageLocation)
-        }
+      // Remove the framework if it exists since it could be out of date.
+      fileManager.removeIfExists(at: zipLocation)
+      fileManager.removeIfExists(at: carthageLocation)
+      do {
+        try fileManager.copyItem(at: framework, to: zipLocation)
+        try fileManager.copyItem(at: framework, to: carthageLocation)
+      } catch {
+        fatalError("Cannot copy framework at \(framework) while " +
+          "attempting to generate frameworks. \(error)")
       }
-      toInstall[podName] = frameworks
-      carthageToInstall[podName] = carthageFrameworks
-    }
-
-    if includeCarthage {
-      return (toInstall, carthageToInstall)
-    } else {
-      return (toInstall, nil)
+      frameworks.append(zipLocation)
+
+      CarthageUtils.generatePlistContents(
+        forName: framework.lastPathComponent.components(separatedBy: ".").first!,
+        withVersion: podInfo.version,
+        to: carthageLocation
+      )
+      carthageFrameworks.append(carthageLocation)
```

Comment:
Does this fail if carthage build is disabled, or does it return the plist contents?

Reply:
This is the binary only path and the return value gets ignored by the caller if Carthage is disabled.